index.md

```diff
@@ -21,10 +21,15 @@ Find good pages about me, on a “more personal” level than the GitHub profile
 - [My programming profile](http://diddileija.github.io/profile). Read a short document about the things I use.
 - [My Python packages](http://diddileija.github.io/py_packages). Read about some of my uploaded Python packages.
 
-Also, find pages on my [projects section](http://diddileija.github.io/projects):
+(If you want a “more general” profile, go to my [GitHub profile](http://github.com/DiddiLeija)).
+
+### Pages on my [projects section](http://diddileija.github.io/projects/)
+
```

Comment:
I was too lazy to open a separate PR, so I made this here, even when it's not related at all :)